desk/app/bait.hoon

```diff
@@ -230,13 +285,73 @@
       %bait-describe
     =+  !<([=nonce:reel =metadata:reel] vase)
     =/  =token:reel  (scot %uv (end [3 16] eny.bowl))
-    :_  this(token-metadata (~(put by token-metadata) token metadata))
+    ::  record the token metadata and add the token to the stable-id set
+    ::  if the group field exists.
+    ::
+    =.  token-metadata
+      (~(put by token-metadata) token metadata)
+    =+  id=(~(get by fields.metadata) 'invitedGroupId')
```

Comment:
Wondering if we should just have a type for this field, that's like `$?(%'invitedGroupId' %'inviterUserId' %'etc')` so that we don't accidentally invent new ones...

Reply:
I have talked with @latter-bolden about this yesterday. We have agreed that it would be best to type up some spec for what we want from lure invites, and then introduce proper types at the backend.

In the meantime, however, we could indeed type those fields as a bandaid.